include/seastar/util/log.hh

```diff
@@ -214,20 +221,23 @@ public:
     /// \param args - args to print string
     ///
     template <typename... Args>
-    void log(log_level level, format_info fmt, Args&&... args) noexcept {
+    void log(log_level level, fmt_string<Args...> fmt, Args&&... args) noexcept {
```

Comment:
This is a public API. So we should retain the old one (perhaps [[deprecated]] with sufficiently new fmt).

Reply:
i see. changed the parameter type of `format_info` to a templated type. so this function won't be found when the compiler is looking for a match for 
```c++
      seastar_logger.log(lvl,
        "IO queue uses {:.2f}ms latency goal for device {}",
        goal.count() * 1000, cfg.devid);
```